packages/pluggableWidgets/maps-native/src/Maps.editorConfig.ts

```diff
@@ -0,0 +1,19 @@
+import { StructurePreviewProps } from "@mendix/piw-utils-internal";
+import { MapsPreviewProps } from "../typings/MapsProps";
+import StructurePreviewMapsSVG from "./assets/StructurePreviewMaps.svg";
+
+export function getPreview(_: MapsPreviewProps): StructurePreviewProps {
+    return {
+        type: "RowLayout",
```

Comment:
I think you don't need the RowLayout. I achieved the same for maps-web by just returning the Image type object.